go.mod

```diff
@@ -1,6 +1,6 @@
 module volcano.sh/volcano
 
-go 1.14
+go 1.16
```

Comment:
Let's use 1.14 until further discussion. :)

Reply:
Sounds good I'll switch this back to 1.14

Reply:
Oh wait I remember why I changed it to 1.16 it's because of the docker containers used for building, let me just verify an amd64 build with changing this back to 1.14 first quickly.

Reply:
Just did the build looks fine, I'll switch this back to 1.14 :)

Reply:
Just did the build looks fine, I'll switch this back to 1.14 :)